src/Microsoft.VisualStudio.Editors/PropPages/ReferencePropPage.vb

```diff
@@ -611,7 +611,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                         Return True
                     End If
 
+#Disable Warning RS1024 ' Compare symbols correctly
```

Comment:
Is this a bug that we're comparing symbols incorrectly? Or is this a false positive hence why you suppressed this?

Reply:
this was detected as a warning on my local build and are converted to errors on Github

Reply:
Yes, it's a bug - either we fix said bug, or we file a bug to fix said bug. Suppressing the warning without an associated bug hides the bug.